packages/cli/src/commands/types/generate/index.test.ts

```diff
@@ -405,5 +406,51 @@ describe('types generate', () => {
         path: undefined,
       });
     });
+
+    it('handles generateTypes returning an array (separate-files mode) without crashing', async () => {
```

Comment:
suggestion: `without crashing` is a little redundant here. Like, obviously, we expect our code not to crash ;)


```suggestion
    it('should pass separateFiles option to generateTypes when --separate-files flag is used', async () => {
```